apps/stix-shifter/CHANGELOG.md

```diff
@@ -11,7 +11,9 @@ Every entry has a category for which we use the following visual abbreviations:
 - ⚡️ Breaking Changes
 - 🐞 Bug Fixes
 
-<!-- ## Unreleased -->
+## Unreleased
+- 🎁 `stix-shifter-threatbus` now comes with its own Dockerfile. Pre-built
+  images are available on Dockerhub.
```

Comment:
Missing a newline.

Please add a link to Dockerhub, e.g., http://hub.docker.com/r/tenzir/stix-shifter-threatbus.